tools/topology/topology2/cavs-passthrough-hda.conf

```diff
@@ -0,0 +1,362 @@
+Define {
+	ANALOG_PLAYBACK_PCM		'Analog Playback'
+	ANALOG_CAPTURE_PCM		'Analog Capture'
+	HDA_ANALOG_DAI_NAME      	'Analog'
+	CODEC_HDA_CHAIN_DMA		'true'
+}
+
+Object.Dai.HDA [
+	{
+		name $HDA_ANALOG_DAI_NAME
+		dai_index 0
+		id 4
+		default_hw_conf_id 4
+		Object.Base.hw_config.1 {
+			name	"HDA0"
+		}
+		direction duplex
+	}
+]
+
+Object.Pipeline {
+
+	host-gateway-playback [
+		{
+			index	0
+			use_chain_dma $CODEC_HDA_CHAIN_DMA
+
+			Object.Widget.host-copier.1 {
+				stream_name $ANALOG_PLAYBACK_PCM
+				pcm_id 0
+				num_output_pins 1
+				IncludeByKey.CODEC_HDA_CHAIN_DMA {
+					"false" {
+					       num_input_audio_formats 3
+					       num_output_audio_formats 3
+				       	       Object.Base.input_audio_format [
```

Comment:
"git log -p" shows some red blocks for this patch, so there's apparently lines those could use tabs instead of spaces.

Reply:
yeah, I have no idea what emacs does with those .conf files. It's probably trying to do something so smart that I don't understand it.